moveit_ros/move_group/src/default_capabilities/execute_trajectory_action_capability.cpp

```diff
@@ -106,8 +106,29 @@ void MoveGroupExecuteTrajectoryAction::executePath(const std::shared_ptr<ExecTra
 {
   RCLCPP_INFO(getLogger(), "Execution request received");
 
+  // Convert mdof trajectory to RobotTrajectory
```

Comment:
This conversion block needs to be added to the Move action as well. Alternatively, we could bring the conversion into the TEM to reduce some redundancy, however that would probably require even more API changes.

Reply:
Maybe we can move it into TEM as some follow-up tasks? That component needs to be cleaned-up heavily anyways

Reply:
done